src/k8s/cmd/k8s/k8s_x_wait_for.go

```diff
@@ -21,12 +21,8 @@ func newXWaitForCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 			ctx, cancel := context.WithTimeout(cmd.Context(), opts.timeout)
 			defer cancel()
 			if err := control.WaitUntilReady(ctx, func() (bool, error) {
-				ok, err := features.StatusChecks.CheckDNS(cmd.Context(), env.Snap)
-				if ok {
-					return true, nil
-				}
-				cmd.PrintErrf("DNS not ready yet: %v\n", err.Error())
```

Comment:
do not drop this message, it's useful to let the user know that things are happening. also below